react-graph/src/Plot.tsx

```diff
@@ -118,8 +133,8 @@ const Plot: React.FunctionComponent<IProps> = (props) => {
     // ToDo: This is hardset to two because it's tied to display, 'left' and 'right'
     const [yHasData, setYHasData] = React.useState<boolean[]>(Array(2).fill(0));
 
-    const [mouseMode, setMouseMode] = React.useState<'none' | 'zoom' | 'pan' | 'select'>('none');
-    const [selectedMode, setSelectedMode] = React.useState<'pan' | 'zoom' | 'select'>('zoom');
+    const [mouseMode, setMouseMode] = React.useState<'none' | 'zoom-rectangular' | 'zoom-vertical' | 'zoom-horizontal' | 'pan' | 'select'>('none');
+    const [selectedMode, setSelectedMode] = React.useState<'zoom-rectangular' | 'zoom-vertical' | 'zoom-horizontal' | 'pan' | 'select'>('zoom-rectangular');
```

Comment:
This matches `GraphContext.CurrentMode` ( + 'none`) Should we move this into a common `type`?